src/loader.js

```diff
@@ -43,9 +43,9 @@ export function loadGooglePlacesLibrary(apiKey, callback) {
   const element = document.createElement('script')
   element.async = true
   element.defer = true
-  element.onload = onLibraryLoaded
-  element.src = `https://maps.googleapis.com/maps/api/js?key=${encodeURIComponent(apiKey)}&libraries=places`
+  element.src = `https://maps.googleapis.com/maps/api/js?key=${encodeURIComponent(apiKey)}&libraries=places&callback=onLibraryLoaded`
   element.type = 'text/javascript'
+  window.onLibraryLoaded = onLibraryLoaded
```

Comment:
In theory, this looks fine. However, I'm a little concerned about the global-ness of `window.onLibraryLoaded`. For example, if there are two instances of this used on a page, will the `callbacks` array accessed by one of the instance's `onLibraryLoaded()` methods have a complete list of the callbacks? It's _probably_ not a problem, but if there were some kind of edge-case bug there, it might be very hard to figure out. (I think it would only be a problem if there were somehow two separate Svelte apps on the same page... which I can't even imagine.)

To avoid that whole scenario, though, what if we left the existing `element.onload = onLibraryLoaded` line and simply passed `callback=Function.prototype` to the Google API, to satisfy it's requirement for a `callback` parameter? (Discussed on StackO [here](https://stackoverflow.com/a/75212692/3813891).)

Reply:
@cptcrunchy Any response to this conversation?